test/LoadTest/cache_refresh/drop_database_mongo.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/
+# Copyright 2015 Telefonica Investigaci�n y Desarrollo, S.A.U
+#
+# This file is part of Short Term Historic (FI-WARE project).
+#
+# iot-sth is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+# iot-sth is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with iot-sth. If not, see
+#    http://www.gnu.org/licenses/.
```

Comment:
The licencse header check script is quite strict, loking for literal matching. Not sure if this will work... I'd recomment to run "make check_file_compliance" to check. If fails, the header in the reqs_x_secs.sh looks better (so you could copy-paste from there).